scripts/benchmarks/README.md

```diff
@@ -37,11 +37,16 @@ By default, the benchmark results are not stored. To store them, use the option
 
 To compare a benchmark run with a previously stored result, use the option `--compare <name>`. You can compare with more than one result.
 
```

Comment:
above there's a sentence `By default, all benchmark runs are compared against `baseline`, which is a well-established set of the latest data.` which should be now gone, I believe.